include/lsst/meas/modelfit/DoubleShapeletPsfApprox.h

```diff
@@ -70,7 +70,7 @@ class DoubleShapeletPsfApproxControl {
     );
 
     LSST_NESTED_CONTROL_FIELD(
-        optimizer, lsst.meas.modelfit.modelfitLib, OptimizerControl,
+        optimizer, lsst.meas.modelfit, OptimizerControl,
```

Comment:
I think that should work, and it would indeed be more consistent with what I've done elsewhere.  I'll give it a try.

I'm afraid all of this control field wrapping is a bit of a fragile mess right now; see comments on [DM-9222](https://jira.lsstcorp.org/browse/DM-9222).  It's not a regression from Swig (it was also a mess before), but I hope pybind11 will give us an opportunity to fix it.